project/Settings.scala

```diff
@@ -19,6 +19,7 @@ object Settings {
     crossScalaVersions := Seq(scala_2_11, scala_2_12),
     releaseCrossBuild := true,
 
+    fork in Test := true,
```

Comment:
why this change?

Reply:
This line fixes tests execution with scala 2.11. It's a Scala issue. Scala links wrong version of scala-parser-combinators.

Reply:
ok, makes sense